Scripts/install_swiftlint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Installs the SwiftLint package.
+# Tries to get the precompiled .pkg file from Github, but if that
+# fails just recompiles from source.
+
+set -e
+
+SWIFTLINT_PKG_PATH="/tmp/SwiftLint.pkg"
+SWIFTLINT_PKG_URL="https://github.com/realm/SwiftLint/releases/download/0.9.1/SwiftLint.pkg"
```

Comment:
Note that I think I borrowed that script from SwiftLint or SourceKitten or somewhere like this, so maybe:

* Worth mentioning the source somewhere
* Worth checking if they have an updated version of this shell script